src/Hl7.Fhir.Base/Model/Generated/Coding.cs

```diff
@@ -121,7 +121,6 @@ public string Version
     /// Symbol in syntax defined by the system
     /// </summary>
     [FhirElement("code", InSummary=true, Order=50)]
-    [CqlElement(IsPrimaryCodePath = true)]
```

Comment:
Should this class not implementing `ICoded` as well?

Reply:
No, ICoded<T> should only be used on resources - this is a mistake in the CQL specification and I have corrected it here.